mono/metadata/domain.c

```diff
@@ -563,7 +562,7 @@ mono_init_internal (const char *filename, const char *exe_filename, const char *
 		 * that would mean it would be reloaded later. So instead, we save it to
 		 * exe_image, and close it during shutdown.
 		 */
-		get_runtimes_from_exe (exe_filename, &exe_image, runtimes);
+		runtimes = get_runtimes_from_exe (exe_filename, &exe_image, runtimes);
```

Comment:
`runtimes` on the RHS here is always NULL, right?

Reply:
Yeah, no reason to pass it at all. Will change.